docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+FROM nvidia/cuda:10.2-base
+
+# Install base dependencies
+RUN apt-get update && \
+    apt-get install -y curl git build-essential wget bzip2 ca-certificates \
+    libglib2.0-0 libxext6 libsm6 libxrender1 
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+SHELL ["/bin/bash", "-c"] 
+
+# Install Anaconda and download the seismic-deeplearning repo
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    apt-get install -y zip && \
+    wget --quiet https://github.com/microsoft/seismic-deeplearning/archive/master.zip -O master.zip && \
+    unzip master.zip && rm master.zip
+
+WORKDIR seismic-deeplearning-master
+
+# RUN cd seismic-deeplearning-master && \
+RUN conda env create -n seismic-interpretation --file environment/anaconda/local/environment.yml && \
+    source activate seismic-interpretation && \
```

Comment:
any reason for not using "conda activate"? I have a workaround (fix) in case this doesn't work

Reply:
I've ran into so many issues with `conda activate`. I think because this is based on Ubuntu 16.04, `source activate` is the default command. If you have any tips on how to get `conda activate` to work, please let me know ;)

Reply:
This worked for me:
```python
# bug to fix conda not launching from a bash shell
source /data/anaconda/etc/profile.d/conda.sh
conda activate seismic-interpretation
```

Reply:
But yeah, source activate is fine too